CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 ### Grafana Mimir
 
+* [CHANGE] The following deprecated configurations have been removed: #6673
+  * `-querier.query-ingesters-within`
```

Comment:
We probably aslo need to update runbook https://github.com/grafana/mimir/blob/main/docs/sources/mimir/manage/mimir-runbooks/_index.md.

Reply:
I think the flag still exists, but it's now a tenant-level override